source/variables/typography.css

```diff
@@ -0,0 +1,13 @@
+:root {
+	/*The base font-size for the root html element.  The value represents its pixel equivalent, e.g. 16 = 16px.*/
+    --base--font-size: 16;
+
+    /*The percentage of the font-size to be used on the root html element.*/
+    --base--font-percentage: calc(calc(var(--base--font-size) / 16) / 100%);
+    
+    /*The base line height for the root html element.*/
+    --base--line-height: 1.5;
+
+    /*The base font-family for the root html element.*/
+    --base--font-family: sans-serif;
+}
```

Comment:
ok update. I'm not a huge fan of having a file called type.css though, seems disconnected from its purpose where as typography.css is hard to not understand.